frontend/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:8
+
+WORKDIR /app
+COPY . ./
+
+RUN npm install --silent
```

Comment:
This works but it's potentially inefficient since container would need to be rebuilt every time some file you copy over from the root directory changes (even if those files don't affect the frontend).

You likely only want to re-run ``npm install`` only when ``package.json`` and any of the frontend files change.

Reply:
True. This would greatly speed up builds, I will check out if there a way to watch for file changes during Docker build.